atlasdb-ete-tests/build.gradle

```diff
@@ -21,6 +21,10 @@ dependencies {
     compile "io.dropwizard:dropwizard-core:" + libVersions.dropwizard
     compile 'com.google.dagger:dagger:2.0.2'
 
+    runtime(project(':atlasdb-dbkvs')) {
+        exclude(module:'HikariCP')
```

Comment:
What is this exclude for?  Are these not the same all using the same Hikari version?